problems/oldestPerson.js

```diff
@@ -11,7 +11,17 @@
  * ex: oldestPerson({ 'Marcus': 29, 'Julia': 29, 'Kevin': 34 })
  * returns: 'Kevin'
  */
-function oldestPerson() {
+const oldestPerson = (obj) => {
+    let string = "";
```

Comment:
To make your code more expressive, you rename this variable to something like `name`. It allows us to read it and understand the type of data it is going to hold